### ISSUE TITLE
TouchUI Mode Compatibility

### ISSUE THREAD

**Issue by brianhanifin** [Peaches491/OctoPrint-Touchtest#5]
I would like this control available in TouchUI mode so that I can control the movement of the bed during leveling from a phone sitting next to my printer, rather than a laptop or iPad off away from my printer.

This would also help the users with a [touch screen attached to their OctoPi rig](url=https://learn.adafruit.com/3-dot-5-pitft-octoprint-rig/overview).

**Comment by cre8nchaos** [comment]
I agree. Would be extremely useful to have this plugin work with touchui

**Comment by FlyingEagle** [comment]
Are there any news of this issue?
